Intersect.Client/Entities/Player.cs

```diff
@@ -130,6 +130,19 @@ public bool IsInMyParty(Entity entity)
             return false;
         }
 
+        public bool IsInMyParty(Guid id)
```

Comment:
I know you're a LINQ lover, and this is a case where I would honestly do this instead:

```csharp
public bool IsInMyParty(Guid id) => Party.Any(member => member.Id == id);
```

Reply:
Had to make additional changes to make this work for both checks.